acai-ios/SectionControllers/OrderCustomizationListSectionController.swift

```diff
@@ -78,17 +96,19 @@ class OrderCustomizationListSectionController: ListSectionController {
         switch type {
         case .base:
             self.selectOptionDelegate?.deselectOption(at: index)
+        case .size:
+            self.selectOptionDelegate?.deselectOption(at: index)
         case .topping:
             return
         }
     }
     
     override func didSelectItem(at index: Int) {
-        guard let type = customizationOptions.options[index].type else { return }
-        if type == .base {
+        guard let type = customizationOptions.options[index - 1].type else { return }
```

Comment:
if `index` is 0, won't this crash

Reply:
I have:
override func numberOfItems() -> Int {
        return customizationOptions.options.count + 1
    } 
because I wanted to group the header with the section so there's no need for a HeaderItem or HeaderSectionController.
I planning to make one detailView for smoothies, drinks, and bowls, but they're still separate bc they're going to have a different number of customization section controllers and having an array for [OrderCustomizationOptions] in the detail view wasn't working with updating the isSelected.